RELEASE_NOTES.md

```diff
@@ -0,0 +1,21 @@
+#### `Attribute service`  (0.9.3 .. 0.9.3)
```

Comment:
Why are we checking in this file?

Reply:
Currently, as we are releasing manually, we can copy paste them. But we will be moving it to automated release once we have self-hosted runner for e2e tests in place. So that time, it will just `cat` this file and paste it in release description. :)